test/commons/color/flatten-colors.js

```diff
@@ -38,9 +38,251 @@ describe('color.flattenColors', function() {
     assert.equal(flat5.alpha, 1);
 
     var flat6 = axe.commons.color.flattenColors(quarterLightGreen, halfRed);
-    assert.equal(flat6.red, 96);
```

Comment:
why did this change?

Reply:
The way were were actually blending the colours was not quite accurate. When you start adding alpha values to colours the way they are calculated is different. Opacity adds another step whereby the pre-multiplied computed new RGB values need to be divided by the new computed alpha, we were missing this step: 

> To extract the color component of a pre-multiplied value, the formula is reversed: Co = co / αo
source: https://www.w3.org/TR/compositing-1/#simplealphacompositing
helpful source: https://github.com/w3c/fxtf-drafts/issues/440#issuecomment-956418953